src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonElement.cs

```diff
@@ -1224,14 +1229,19 @@ internal ReadOnlySpan<byte> ValueSpan
             }
         }
 
-        // TODO make public https://github.com/dotnet/runtime/issues/33388
-        internal static bool DeepEquals(JsonElement left, JsonElement right)
+        /// <summary>
+        /// Compares the values of two <see cref="JsonElement"/> values for equality, including the values of all descendant elements.
```

Comment:
Consider saying something about ordering in remarks.

Reply:
I've added a remarks section with an informal specification of deep equality.